docs/conf.py

```diff
@@ -82,7 +85,7 @@
 
 # General information about the project.
 project = 'Siphon'
-copyright = '2015, Unidata'
+copyright = '2018, Unidata'
```

Comment:
Following Unidata/siphon#198, should this be changed from "Unidata" to "Siphon Contributors"?

Reply:
It should be, but I did all that in the last PR...

Reply:
Yep - this was mostly done yesterday afternoon before you did all that. I'll kill that commit.